Examples/Statistics/BayesianClassifier.cxx

```diff
@@ -75,10 +75,7 @@ main(int argc, char * argv[])
   constexpr unsigned int Dimension = 2;
   using InputPixelType = float;
   using InputImageType = itk::VectorImage<InputPixelType, Dimension>;
-  using ReaderType = itk::ImageFileReader<InputImageType>;
-
-  auto reader = ReaderType::New();
-  reader->SetFileName(membershipImageFileName);
+  const auto input = itk::ReadImage<InputImageType>(membershipImageFileName);
```

Comment:
I just realized that this might throw. The [testing macro `ITK_TRY_EXPECT_NO_EXCEPTION`](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.4.0/Modules/Core/TestKernel/include/itkTestingMacros.h#L130-L142) is unsuitable for use in examples. We should probably put the entire code inside try/catch block. Alternatively use this pattern:
```c++
using ImageType::Pointer input;
try
{
  input = itk::ReadImage<ImageType>(argv[1]);
}
catch (const itk::ExceptionObject & excp)
{
  std::cerr << excp << std::endl;
  return EXIT_FAILURE;
}
```
This pattern is already used in some example. But this can be a follow-up PR.